@xen-orchestra/web-core/lib/components/menu/MenuList.vue

```diff
@@ -1,11 +1,20 @@
 <!-- v1.0 -->
 <template>
-  <slot :is-open="isOpen" :open="open" name="trigger" />
-  <Teleport :disabled="!shouldTeleport" to="body">
-    <ul v-if="!hasTrigger || isOpen" ref="menu" :class="{ horizontal, shadow }" class="menu-list" v-bind="$attrs">
+  <slot :is-open="isOpen" :open="open" name="trigger" :is-header="false" />
```

Comment:
You can use the shorthand here:
```suggestion
  <slot :is-open="isOpen" :open name="trigger" :is-header="false" />
```